scripts/train_expert_policy.py

```diff
@@ -9,6 +9,51 @@
 from garage.tf.algos import TRPO
 from garage.tf.policies import CategoricalMLPPolicy
 
+from akro.discrete import Discrete
+
+# These wrappers are inspired by https://github.com/maximecb/gym-minigrid/blob/master/gym_minigrid/wrappers.py
+class WithActionObs(gym.core.ObservationWrapper):
```

Comment:
This isn't being used right? Should we remove to keep code simple? But keep the inspiration comment?

Reply:
Yes, it is not used. I can remove it